admin-dev/themes/new-theme/scss/pages/_product_page.scss

```diff
@@ -577,6 +577,14 @@
     }
   }
 
+  .maxLength {
+    float: right;
+    font-family: OpenSans;
+    font-size: 10px;
```

Comment:
10px is very very small nope ?
![lea](https://cloud.githubusercontent.com/assets/9074610/21585967/e0b4b180-d0ca-11e6-92bf-d5e5bced061d.png)

ping @Leadesign